en/simulation/hitl.md

```diff
@@ -26,7 +26,14 @@ With Hardware-in-the-Loop (HITL) simulation the normal PX4 firmware is run on re
 
 ### JMAVSim/Gazebo HITL Environment
 
-JMAVSim or Gazebo (running on a development computer) are connected to the flight controller hardware via USB/UART. The simulator acts as gateway to share MAVLink data between PX4 and *QGroundControl*. 
+JMAVSim or Gazebo (running on a development computer) are connected to the
```

Comment:
This is a bit complex to parse, and no longer "quite" reflects the diagram. How about if we pull out the RPi stuff into a note?

How about:

----

JMAVSim or Gazebo (running on a development computer) are connected to the flight controller hardware via USB/UART. The simulator acts as gateway to share MAVLink data between PX4 and *QGroundControl*. 

> **Note** The simulator/flight controller can also be connected via UDP if the flight controller has networking support and uses a stable, low-latency connection (e.g. a wired Ethernet connection - WiFi is not usually sufficiently reliable).  For example, this configuration has been tested using Raspberry Pi via Ethernet (a startup configuration that includes the command for running jMAVSim can be found [here](https://github.com/PX4/Firmware/blob/master/posix-configs/rpi/px4_hil.config)).

Reply:
I almost did it that way but then I thought we start to have too many Notes and Tips there.